state/podspec_ops.go

```diff
@@ -73,9 +73,9 @@ func (op *setPodSpecOperation) buildTxn(_ int) ([]txn.Op, error) {
 	if err != nil {
 		return nil, errors.Annotate(err, "setting pod spec")
 	}
-	if app.Life() != Alive {
+	if app.Life() == Dead {
```

Comment:
This is definitely the only place where we do the check?
Nothing in the unit agent?

Reply:
Nah